backend/functions/lib/humble-objects/mongo-humble.ts

```diff
@@ -0,0 +1,71 @@
+import { Collection, Db, MongoClient } from 'mongodb';
+import { DocumentQuery, transformQuery } from '../adapters/gateways/document-db.repository';
+import { Closable } from '../defer-close';
+
+export class CollectionHumble<T> {
+  private collection: Collection<T>;
+
+  constructor(database: Db, collectionName: string) {
+    this.collection = database.collection<T>(collectionName);
+  }
+
+  public async find(query: DocumentQuery) {
+    return this.collection.find(transformQuery(query));
```

Comment:
See notes from the `document-db.respoitory.ts` file about how the `transformQuery` function is redundant no longer necessary with the query builder and the mongo adapter.